third_party/iap_jwt_verify_nginx/src/jwt.cc

```diff
@@ -0,0 +1,209 @@
+// Copyright (C) 2002-2016 Igor Sysoev
```

Comment:
Update: apparently I was Googling slightly-wrong search terms all along because it seems there are a couple decent options I didn't find until just now:

https://github.com/latchset/jose
https://github.com/cisco/cjose

Neither one is used internally at Google AFAICT, so they still don't meet my bar for "I'm comfortable pulling these into a security feature without extensive review". We could certainly considering migrating to one (most likely the Cisco one, it's endorsed by OpenID).